package.json

```diff
@@ -14,6 +14,7 @@
     "react-router-dom": "^6.7.0",
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214757925-b60d43f4-7dbf-448e-83a8-ab5d19e48980.png)

- [ ] In your license file you should add the current year which is 2023, not 2021 :smile: 
- [ ] Kindly add **[APP TYPE]** in your license file such as **web app** or **website** to make your project licensed :+1: